conf/nginx-top.conf

```diff
@@ -7,6 +7,6 @@
 ##       your own --- please do not ask for help from us.
 
 upstream php-fpm {
-	server unix:/var/run/php/php7.2-fpm.sock;
+	server unix:/var/run/php/php8.0-fpm.sock;
```

Comment:
There are some instances that have ...php8.0... and some that have php$PHP_VER, should this be unified to also use `phpPHP_VER`?

Reply:
$PHP_VER is defined in the setup bash scripts. These other files don't have access to that variable, and since there were only a few, I thought it was easier to leave them as it is rather than add new variable declarations/substitution logic.